crates/burn-import/src/burn/node/argmax.rs

```diff
@@ -0,0 +1,108 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{TensorKind, TensorType, ToTokens, Type};
+
+use burn::record::PrecisionSettings;
+use quote::quote;
+
+#[derive(Debug, Clone, new)]
+pub struct ArgMaxNode {
+    pub input: TensorType,
+    pub output: TensorType,
+    pub axis: usize,
+    pub select_last_index: usize,
+    pub keepdims: usize,
+}
+
+impl<PS: PrecisionSettings> NodeCodegen<PS> for ArgMaxNode {
+    fn output_types(&self) -> Vec<Type> {
+        let mut output = self.output.clone();
+        output.kind = TensorKind::Int;
+        vec![Type::Tensor(output)]
+    }
+
+    fn input_types(&self) -> Vec<crate::burn::Type> {
+        vec![Type::Tensor(self.input.clone())]
+    }
+
+    fn forward(
+        &self,
+        scope: &mut crate::burn::Scope,
+        node_position: usize,
+    ) -> proc_macro2::TokenStream {
+        let axis = self.axis.to_tokens();
+
+        //NOTE: are select_last_index and keep_dims supported?
+        let _select_last_index = self.select_last_index.to_tokens();
+        let _keepdims = self.keepdims.to_tokens();
```

Comment:
`select_last_index` is not supported in Burn, so we could add a warning when checking the node config if it is set.

Same for `keep_dims`, the argmax method in Burn returns a tensor of the same shape as the input. But if the tensor has more than 1 dimension, we could generate the code to get the same output as the ONNX node by doing `argmax` followed by a squeeze on the selected axis. For this PR, feel free to keep the support limited to `keep_dims` set to 1 (and panic when checking the node config if it is 0) or add the sequence of ops with a squeeze after the argmax.

And then you can remove the unused fields from the codegen here.